doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -51,7 +51,10 @@ For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
 Bluetooth mesh
 --------------
 
-|no_changes_yet_note|
+* Changed UUID generation in the module mesh provisioning handler for Nordic DKs. The change prevents trailing zeros in the UUID.
```

Comment:
Also wondering if this should be placed under Libraries/Bluetooth mesh profile instead?